Sources/Brave/Frontend/Settings/Tabs/PrivateTabsView.swift

```diff
@@ -86,7 +89,15 @@ struct PrivateTabsView: View {
         case .faceID, .touchID, .pinCode:
           OptionToggleView(title: browsingLockTitle,
                            subtitle: nil,
-                           option: Preferences.Privacy.privateBrowsingLock)
+                           option: privateBrowsingLock) { newValue in
+            if !newValue {
+              askForAuthentication(.general) { success, _ in
+                if success {
+                  privateBrowsingLock.value = newValue
+                }
+              }
```

Comment:
Brought up in DM but this will still update the preference immediately and currently does not reset it back to false if auth fails. Likely need to replace this with the underlying view that gives you access to a Binding, then create a custom Binding that doesn't set the pref to false at all until auth completes

Reply:
Thanks @kylehickinson , good catch! I tried this with `Browser lock` switch and it didn't work, but didn't check `Private tabs`.

Reply:
Fixed so the preference only changes after authentication.